packages/larva-patterns/modules/pullquote/pullquote.extended.js

```diff
@@ -0,0 +1,8 @@
+const clonedeep = require('lodash.clonedeep');
+
+const pullquote_prototype = require( './pullquote.prototype' );
+const pullquote_extended = clonedeep( pullquote_prototype );
+
+pullquote_extended.pullquote_classes += ' extended';
+
+module.exports = pullquote_extended;
```

Comment:
There is an error in this file, it needs a blank line.

Reply:
Resolved, not sure why it did not flagged in `npm run lint`